### ISSUE TITLE
[a11y]: Export Modal, Focus not returning to triggering button

### ISSUE THREAD

**Issue by devadula-nandan** [carbon-design-system/ibm-products#6018]
### Package

@carbon/ibm-products

### Browser

Firefox

### Operating System

Windows

### Package version

v2.49.0-rc.0

### Automated testing tool and ruleset

manual

### Assistive technology

JAWS

### Description

Reported by Raghu during QA audit.

Focus not returning to "Launch modal" button on closing the Export dialog

Once the Modal closed , focus expected to set back to the Triggering  control of the Modal , in this case Triggering control is Launch modal button

After a user action, if the focus randomly jumps to another spot on the web page, it’s tedious to navigate back to the relevant element. This negatively affects the experience of keyboard-only users and those with cognitive disabilities.

### WCAG 2.1 Violation

2.4.3 Focus Order

### Reproduction/example

https://ibm-products.carbondesignsystem.com/?path=/story/ibm-products-components-export-exportmodal--standard&globals=viewport:basic

### Steps to reproduce

open the export modal
close it, and notice the focus is not returning to the triggering button

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems
